ckanext/xloader/tests/test_action.py

```diff
@@ -117,10 +118,24 @@ def test_status(self):
 
         assert status["status"] == "pending"
 
-    def test_xloader_user_api_token_defaults_to_site_user_apikey(self):
-        api_token = get_xloader_user_apitoken()
-        site_user = helpers.call_action("get_site_user")
-        assert api_token == site_user["apikey"]
+
+    def test_xloader_user_api_token_from_config(self):
+        sysadmin = factories.SysadminWithToken()
+        apikey = sysadmin["token"]
+        with mock.patch.dict(toolkit.config, {'ckanext.xloader.api_token': apikey}):
```

Comment:
this is to mimic what would be in the config.  I ran out of time to work out how to look up a key inside pytest or hope that the db setup at the start was not 'cleansed'

Reply:
For dynamic token values this approach does make sense.